great_expectations/core/expectation_suite.py

```diff
@@ -144,8 +144,12 @@ def __init__(  # noqa: PLR0913
         self._store = project_manager.get_expectations_store()
 
     @property
-    def name(self) -> str:
-        return self.expectation_suite_name
+    def expectation_suite_name(self) -> str:
+        return self.name
+
+    @expectation_suite_name.setter
+    def expectation_suite_name(self, value) -> None:
+        self.name = value
```

Comment:
Can we just remove this since we're open to breaking changes?

Reply:
yes! was going to scope that to a different PR, since its significantly more involved